libsolidity/ast/Types.h

```diff
@@ -355,6 +355,19 @@ class IntegerType: public Type
 	Modifier m_modifier;
 };
 
+struct ElementaryTypes
+{
+	static IntegerType Address;
+	// static const ArrayType BytesMemory;
+	// static const ArrayType StringMemory;
+	// ElementaryTypes()
+	// {
+	// 	Address = IntegerType(160, IntegerType::Modifier::Address);
+	// };
+};
+IntegerType ElementaryTypes::Address = IntegerType(160, IntegerType::Modifier::Address);
```

Comment:
This actually needs to be used the code.

Also it shouldn't be instantiated in the header.